minecraft/listener.go

```diff
@@ -43,6 +43,9 @@ type ListenConfig struct {
 	// This field should not be edited during runtime of the Listener to avoid race conditions. Use
 	// Listener.AddResourcePack() to add a resource pack after having called Listener.Listen().
 	ResourcePacks []*resource.Pack
+	// BiomeDefinitions is a list of encoded biome definitions that the server may use. If these are nil,
+	// the default biome definitions will be used.
+	BiomeDefinitions []byte
```

Comment:
I'd make this a map[string]interface{} and leave the NBT encoding to gophertunnel

Reply:
Changed.